pkg/scalers/postgres_scaler.go

```diff
@@ -18,26 +18,26 @@ const (
 	defaultPostgresPassword = ""
 )
 
-type postGRESScaler struct {
-	metadata   *postGRESMetadata
+type postgreSQLScaler struct {
+	metadata   *postgreSQLMetadata
 	connection *sql.DB
 }
 
-type postGRESMetadata struct {
-	connStr  string
-	userName string
-	password string
-	host     string
-	port     string
-	query    string
-	dbName   string
-	sslmode  string
+type postgreSQLMetadata struct {
+	connection string
+	userName   string
+	password   string
+	host       string
+	port       string
+	query      string
+	dbName     string
+	sslmode    string
 }
 
 var postgresLog = logf.Log.WithName("postgres_scaler")
```

Comment:
please rename this as well to keep things consistent, `postgresLog` `postgresLog`